docs/book/README.md

```diff
@@ -23,30 +23,30 @@ You must also install forc plugins that are already documented within the book.
 ```sh
 cargo install --path ./forc-plugins/forc-fmt
 cargo install --path ./forc-plugins/forc-lsp
-cargo install --path ./forc-plugins/forc-explore
+cargo install forc-explore
 cargo install --path ./forc-plugins/forc-doc
 ```
 
 To build book:
 
 ```sh
-mdbook build
+mdbook build # path to book.toml
```

Comment:
As mentioned [here](https://github.com/FuelLabs/sway/issues/3981#issuecomment-1416776740), the instruction to build without the path isn't incorrect, perhaps just poorly explained, since the command assumes the current working directory contains the `book.toml` by default. 

Perhaps we could explain this instead?
```suggestion
To build book, you may either:

1) cd into book directory

cd docs/book && mdbook build

or 

2) execute from the Sway project root

mdbook build docs/book
```

Something along these lines.

Reply:
I think just making the commands specific to our use case is sufficient so my vote goes to just updating the command to `mdbook build docs/book`